prime-router/src/main/kotlin/fhirengine/translation/hl7/schema/converter/ConverterSchema.kt

```diff
@@ -22,6 +22,11 @@ class ConverterSchema(
     constants: SortedMap<String, String> = sortedMapOf(),
     extends: String? = null
 ) : ConfigSchema<ConverterSchemaElement>(elements = elements, constants = constants, extends = extends) {
+
+    override fun toString(): String {
```

Comment:
While debugging I got stick of seeing `...ConverterSchema@194fa1`, so I had a toString to make it easier to understand what element you're looking at.